src/ia/js/IAIndex.js

```diff
@@ -271,6 +270,8 @@
             var selector_all;
             var text_all;
             var tot_count = 0;
+
+            $("#ia_index_header h2").text("Showing " + $(".ia-item:visible").length + " Instant Answers");
```

Comment:
Actually we can look at things like that in a perf pass later. would rather move forward with functionality right now.